addons/account/views/account_view.xml

```diff
@@ -1425,7 +1425,7 @@
             </field>
         </record>
         <record id="view_account_move_line_filter" model="ir.ui.view">
-            <field name="name">Journal Items</field>
+            <field name="name">account.move.line.search</field>
```

Comment:
this is unexpected. Any good reason to do so?

It might look harmless but, if I'm not wrong, changing the name of a view might also change the value of it in the breacrumb if the action has no name. So it isn't that safe.

and I like to keep the diff minimal

Reply:
It is the name of the search view, every other search view in the file has a name like that, I just thought that it is painful to grep in the file, but I'll change it if it really has side effects